Makefile

```diff
@@ -6,69 +6,49 @@ style:
 
 batch_size := 1
 
-install-mqa-transformers:
-	git clone https://github.com/bigcode-project/transformers.git; \
-	cd transformers; \
-	git checkout mayank/multi_query; \
-	pip install .; \
-	cd ..; \
-	rm -rf transformers;
+install:
+	git submodule update --init
+	pip install -r requirements.txt
 
 # BLOOM AliBi
 hf-1b-bloom-fp32:
-	python src/main.py --hidden_size 2048 --n_head 16 --n_layer 24 --pipeline_class HF_GPU_Pipeline --model_class BLOOM --dtype float32 --batch_size ${batch_size}
+	python3 src/main.py --hidden_size 2048 --n_head 16 --n_layer 24 --pipeline_class HF_Pipeline --model_class BLOOM --dtype float32 --batch_size ${batch_size}
```

Comment:
@mayank31398 I don't see why we should have it as a module. It's a script and not installed as a package.